dotnet/src/webdriver/WebDriver.cs

```diff
@@ -854,6 +851,15 @@ private static void UnpackAndThrowOnError(Response errorResponse, string command
                         case WebDriverResult.InsecureCertificate:
                             throw new InsecureCertificateException(errorMessage);
 
+                        case WebDriverResult.UnknownError:
+                            throw new UnknownErrorException(errorMessage);
+
+                        case WebDriverResult.UnknownMethod:
+                            throw new UnknownMethodException(errorMessage);
+
+                        case WebDriverResult.UnsupportedOperation:
+                            throw new UnsupportedOperationException(errorMessage);
+
```

Comment:
There are 3 remaining (in-spec) `WebDriverResult` values which do not have custom exceptions (every single other case does, so I assume it's desirable)

- [`NoSuchCookie`](https://www.w3.org/TR/webdriver2/#dfn-no-such-cookie) - as discussed in https://github.com/SeleniumHQ/selenium/issues/14884 this error code is never returned. The breaking change PR https://github.com/SeleniumHQ/selenium/pull/14883 does introduce the new exception type, but it will not be merged anytime soon. Would it be OK to add just the `NoSuchCookieException` in this PR instead?
- [`SessionNotCreated`](https://www.w3.org/TR/webdriver2/#dfn-session-not-created)
- [`UnableToCaptureScreen`](https://www.w3.org/TR/webdriver2/#dfn-unable-to-capture-screen)

Reply:
> [NoSuchCookie](https://www.w3.org/TR/webdriver2/#dfn-no-such-cookie) - as discussed in https://github.com/SeleniumHQ/selenium/issues/14884 this error code is never returned. The breaking change PR https://github.com/SeleniumHQ/selenium/pull/14883 does introduce the new exception type, but it will not be merged anytime soon. Would it be OK to add just the NoSuchCookieException in this PR instead?

As I understand everybody is OK to use special endpoint to get a named cookie. Then we can do it later.

Reply:
> [SessionNotCreated](https://www.w3.org/TR/webdriver2/#dfn-session-not-created) - I tried to reproduce this based on what the [spec](https://www.w3.org/TR/webdriver2/#dfn-new-sessions) says should return this, and I got a spec violation! I set capabilities to null and got invalid argument instead of the spec-required session not created. I think maybe this error doesn't matter much.
[UnableToCaptureScreen](https://www.w3.org/TR/webdriver2/#dfn-unable-to-capture-screen) - struggling to reproduce this. This may be an extremely edge case scenario.

Let's skip, this PR is making selenium better, enough.